scalecodec/type_registry/polymesh.json

```
@@ -1439,8 +1420,28 @@
         ["price", "Balance"]
       ]
     },
-    "CompactAssignments": "CompactAssignmentsFrom258",
-    "RefCount": "RefCount",
-    "secondary_key::api::LegacyPermissions": "LegacyPermissions"
+    "AssetMigrationError": {
+      "type": "enum",
+      "type_mapping": [
+        ["AssetDocumentFail", "(Ticker, DocumentId)"]
+      ]
+  },
+  "MigrationError": {
+    "type": "enum",
+    "type_mapping": [
+      ["DecodeKey", "Vec<u8>"],
+      ["Map", "AssetMigrationError"]
+    ]
+  },
+  "PermissionedIdentityPrefs": {
+    "type": "struct",
+    "type_mapping": [
+      ["intended_count", "u32"],
+      ["running_count", "u32"]
+    ]
+  },
+  "CompactAssignments": "CompactAssignmentsFrom258",
+  "RefCount": "RefCount",
+  "secondary_key::api::LegacyPermissions": "LegacyPermissions"
```

I found a problem recently solved by adding (I'm not sure why): 

```suggestion
  "secondary_key::api::LegacyPermissions": "LegacyPermissions",
  "secondary_key::api::secondarykey<accountid>": "SecondaryKey"
```